e2e/tests/admin/user-admin-ui.spec.ts

```diff
@@ -0,0 +1,99 @@
+import UserAdminPage from 'app/page/admin-user-list-page';
+import { signInWithAccessToken } from 'utils/test-utils';
+import navigation, { NavLink } from 'app/component/navigation';
+import AdminTable from 'app/component/admin-table';
+import UserProfileInfo from 'app/page/admin-user-profile-info';
+//import UserAuditPage from 'app/page/admin-user-audit-page';
+
+
+describe('Admin', () => {
+  const userEmail = 'admin_test@fake-research-aou.org';
+  // const userName = "admin_test";
+
+  beforeEach(async () => {
+    await signInWithAccessToken(page);
+    //await navigation.navMenu(page, NavLink.ADMIN);
+    await navigation.navMenu(page, NavLink.USER_ADMIN);
+  });
+
+  test('verify user bypassed and status is active', async () => {
+    const userAdminPage = new UserAdminPage(page);
+    await userAdminPage.waitForLoad();
+
+    //look up for the user
+    await userAdminPage.searchUser(userEmail);
+    await userAdminPage.waitForLoad();
+
+    //Verify table column names match.
+    const columns = [
+      'Status',
+      'Institution',
+      'Registration date',
```

Comment:
FYI I'm planning to remove this soon, so let's see if the test catches it!

Reply:
Yes, it'll surely catch the change if the test case is merged to master before the column update. Thanks!